students/alexLaws/lesson03/mailroom.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+donor_list = []
```

Comment:
This could be written as a list of dict literals rather than making a call to `append` for each item in the list (something like this):

```
donor_list = [
    {'Name': 'Ryan Moore', 'Donation': 500},
    {'Name': 'Ryan Moore', 'Donation': 250},
    {'Name': 'Ted Laws', 'Donation': 1000},
 ]
```